src/Info.ts

```diff
@@ -176,4 +199,40 @@ export default class Info {
         info.counter++;
         return info;
     }
+
+    private levenshteinDistance(a: string, b: string): number {
+        if (a === b) {
+            return 0;
+        }
+
+        if (a.length === 0) {
+            return b.length;
+        }
+
+        if (b.length === 0) {
+            return a.length;
+        }
+
+        let v0 = [];
+        const v1 = [];
```

Comment:
one const, one let?

Reply:
v0 is changed later in the loop, so it cant be const.